### PR TITLE
Google calendar integration

### PR DESCRIPTION
**Pull request by michaelsheets88**
#37 
I dont think there is much else to be done for this, besides the admin settings within the Google account, and populating the calendar.  As of right now the calendar shows "Busy" or "Free" rather than any information about the events.

- [x] Change settings in Google Calendar to adjust visibility of events.

screencapture

### CODE REVIEW

**Comment by sorianog** [comment]
@michaelsheets88 Are the admin settings preventing the event details to show up?

File: _events/eventsRedirect.md [hunk 1]
```diff
@@ -6,5 +6,7 @@ permalink: /events/
 
 ## SSE Events
 
+Hi there & welcome to our events page! SSE is proud to sponsor a number of events for MSOE SE and CE students. Check out our hack-n-tells, welcome week activities, company tours and mock-whiteboard interviews! From social networking to professional preparation, SSE hosts your destination events for professional, technical and social networking!
```
Comment (sorianog) [inline _events/eventsRedirect.md]: Ooo, hack-n-tells like RokkinCat's? https://www.rokkincat.com/hack-n-tell/ 
If so, would it make sense to hyperlink that text to their hack-n-tell site?

Comment (GFLEMING133) [reply _events/eventsRedirect.md]: Exactly. We've had groups of SSE people attend their Hack-n-tells before, so linking to their site is a really great idea!

Comment (michaelsheets88) [reply _events/eventsRedirect.md]: I can add that link in there in this PR, or should we make a ticket to scan through the content and add appropriate links everywhere?

**Comment by michaelsheets88** [comment]
> @michaelsheets88 Are the admin settings preventing the event details to show up?

@sorianog Yes, the admin need to go into their calendar settings and make everything public.

File: _events/eventsRedirect.md (hunk 1 above)
Comment (sorianog) [reply _events/eventsRedirect.md]: Add the link here in the PR and for future content you can reference this work as an example. @GFLEMING133 thoughts?

**Pull request merged**